bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -141,60 +177,87 @@ final class BagsManager(executorService: ExecutorService) {
 
         val key = obs.getNodeKey
         state += key
+        if (bagsStatus(key).isEmpty)
+          setStatus(key, BagsStatus.NewPending.some)
+
         Future {
+          // Wait the delay before running.
+          Thread.sleep(delay)
```

Comment:
Sorry, why do we need the `delay`? You mention a chance to recover, recover from what?